stepactions/bundles/pick-cluster-version/0.1/pick-cluster-version.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: tekton.dev/v1beta1
+kind: StepAction
+metadata:
+  name: pick-cluster-version
+spec:
+  description: >-
+    This StepAction selects a cluster version for provisioning. It determines the target OCP version of an FBC fragment.
+    The version is returned if it is included in the list of supported Hypershift cluster versions.
+  image: quay.io/konflux-ci/konflux-test:v1.4.17@sha256:36d3a2a6f4f96df41fc56c7c1c5a11c5d1c621f0f767607fda3f83cc21f638cc
```

Comment:
we have a 1.4.18 release now

Reply:
though, it might not have the changes required for this stepaction. Please consider this suggestion optional

Reply:
thanks, I changed to a 1.4.18 image